src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -536,7 +536,14 @@ export class SkyLookupComponent
 
   public onSearchButtonClick(): void {
     this.sendAutocompleteMessage(SkyAutocompleteMessageType.CloseDropdown);
-    this.openPicker(this.autocompleteInputDirective.inputTextValue);
+
+    let isValueInTextBox = false;
+    if (this.selectMode === SkyLookupSelectMode.single) {
+      isValueInTextBox = this.autocompleteInputDirective.inputTextValue === this.autocompleteInputDirective.value[this.descriptorProperty];
+    }
+
+    this.openPicker(isValueInTextBox ? undefined : this.autocompleteInputDirective.inputTextValue);
```

Comment:
The `openPicker` method expects a `string` value for the `initialSearch` parameter. Setting this to `undefined` might work today, but when we convert to "strict" mode for SKY UX 6 this will cause a problem. Can you see a use-case for the `initialSearch` value being `undefined | string`? If not, maybe we open the picker with an empty string instead (`''`).

Reply:
I can't think of one off the top of my head. Updated to send an empty string instead of undefined.